test/helpers/kubectl.go

```diff
@@ -700,6 +700,7 @@ func (kub *Kubectl) GatherLogs() {
 		"kubectl get ds -o wide --all-namespaces":       "ds.txt",
 		"kubectl get cnp --all-namespaces":              "cnp.txt",
 		"kubectl describe pods --all-namespaces":        "pods_status.txt",
+		"kubectl -n kube-system logs -l k8s-app=cilium": "cilium_logs.txt",
```

Comment:
Looks fine, but what will this look like when there are multiple pods on the system? Will there be something prefixed to the logs to delineate from which pod the logs are from?